blueprints/codes/eurocode/nen_en_1993_1_1_c2_a1_2016/chapter_6_ultimate_limit_state/formula_6_18_sub_av.py

```diff
@@ -0,0 +1,548 @@
+        tf : MM
+            [$t_f$] Flange thickness [$mm$].
+        """
+        super().__init__()
+        self.a = a
+        self.b = b
+        self.tf = tf
+
+    @staticmethod
+    def _evaluate(
+        a: MM2,
+        b: MM,
+        tf: MM,
+    ) -> MM2:
+        """Evaluates the formula, for more information see the __init__ method."""
+        raise_if_negative(a=a, b=b, tf=tf)
+
+        return 0.9 * (a - b * tf)
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 6.18subc."""
+        _equation: str = r"0.9 \cdot (A - b \cdot t_f)"
+        _numeric_equation: str = latex_replace_symbols(
+            _equation,
+            {
+                r"A": f"{self.a:.3f}",
+                r"b": f"{self.b:.3f}",
+                r"t_f": f"{self.tf:.3f}",
+            },
+            False,
+        )
+        return LatexFormula(
+            return_symbol=r"A_v",
+            result=f"{self:.3f}",
+            equation=_equation,
+            numeric_equation=_numeric_equation,
+            comparison_operator_label="=",
+            unit="mm^2",
+        )
+
+
+class Form6Dot18SubDWeldedIHandBoxSection(Formula):
+    r"""Class representing formula 6.18subd for the calculation of shear area for welded I, H, and box sections with load parallel to web."""
+
+    label = "6.18subd"
+    source_document = NEN_EN_1993_1_1_C2_A1_2016
+
+    def __init__(
+        self,
+        hw: list[MM],
+        tw: list[MM],
+        eta: DIMENSIONLESS,
+    ) -> None:
+        r"""[$A_v$] Calculation of the shear area for welded I, H, and box sections with load parallel to web [$mm^2$].
+
+        NEN-EN 1993-1-1+C2+A1:2016 art.6.2.6(3) - Formula (6.18subd)
+
+        Parameters
+        ----------
+        hw : list[MM]
+            [$h_w$] List of depths of the web [$mm$].
+        tw : list[MM]
+            [$t_w$] List of web thicknesses [$mm$]. If the web thickness is not constant, tw should be taken as the minimum thickness.
+        eta : DIMENSIONLESS
+            [$\eta$] See EN 1993-1-5. Note, $eta$ may be conservatively taken equal to 1.0.
+        """
+        super().__init__()
+        self.hw = hw
+        self.tw = tw
+        self.eta = eta
+
+    @staticmethod
+    def _evaluate(
+        hw: list[MM],
+        tw: list[MM],
+        eta: DIMENSIONLESS,
+    ) -> MM2:
+        """Evaluates the formula, for more information see the __init__ method."""
+        raise_if_negative(eta=eta)
+        for h, t in zip(hw, tw):
+            raise_if_negative(h=h, t=t)
+        raise_if_lists_differ_in_length(hw=hw, tw=tw)
+
+        return eta * sum(h * t for h, t in zip(hw, tw))
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 6.18subd."""
+        _equation: str = r"\eta \cdot \sum (h_{w0} \cdot t_{w0}"
+        for i in range(1, len(self.hw)):
+            _equation += rf" + h_{{w{i}}} \cdot t_{{w{i}}}"
+        _equation += ")"
```

Comment:
I think `equation` should be simply: 
![image](https://github.com/user-attachments/assets/3fea737b-841a-4ebe-9f17-6f585e8a4024)

So: 
```suggestion
        _equation = r"\eta \cdot \sum (h_{w} \cdot t_{w})"
```